src/AmbientSounds/ViewModels/SettingsViewModel.cs

```diff
@@ -61,6 +63,33 @@ public bool ScreensaverEnabled
             }
         }
 
+
+        /// <summary>
+        /// Settings flag for screensaver.
+        /// </summary>
+        public bool OverrideLanguage
+        {
+            get => _userSettings.Get<bool>(UserSettingsConstants.OverrideLanguage);
+            set => _userSettings.Set(UserSettingsConstants.OverrideLanguage, value);
+        }
+
+        /// <summary>
+        /// Settings flag for screensaver.
+        /// </summary>
+        public string PreferredLanguage
+        {
+            get => _userSettings.Get<string>(UserSettingsConstants.PreferredLanguage);
+            set => _userSettings.Set(UserSettingsConstants.PreferredLanguage, value);
+        }
+
+
+        //Todo - save it somewhere else?
+        public List<string> AvailableLanugages
+        {
+            get => new List<string>() { "ar", "ca-ES", "cs-CZ", "da-DK", "de-DE", "en-Us", "es-ES", "fr-FR", "hu-HU", "it-IT", "ko-KR", "nl-BE", "pt-BR", "tr" };
```

Comment:
Can you find a way to programmatically fetch the languages stored in the app? This hardcoded list will lead to problems whenever the list of available language changes